src/store/configureStore.ts

```diff
@@ -152,6 +141,18 @@ export default function configureStore(
   // https://github.com/elgerlambert/redux-localstorage/issues/42
   // createPersistentStore should ONLY take reducer and initialState
   // any store enhancers must be added to the compose() function.
-  _store = createPersistentStore(rootReducer(history), initialState)
-  return _store
+  return createPersistentStore(rootReducer(history), initialState)
+}
+
+let storeSingleton
```

Comment:
I'm curious why the storeSingleton is scoped outside the `getStore` function

Reply:
How do you propose to define it within `getStore`?

Reply:
Ya, no. I see why now. Singleton 🎉